base/applications/rapps/include/appview.h

```diff
@@ -20,7 +20,7 @@
 
 using namespace Gdiplus;
 
-#define LISTVIEW_ICON_SIZE 24
```

Comment:
may I ask why this is changed 🤣

Reply:
According to the [Oleg's comment in VK](https://vk.com/wall-1086956_82280?reply=82283&thread=82282), he changed it because almost all existing apps provide either 16x16 or 32x32 icons. RAPPS itself used 24x24 icon which was the same for all entries, and when we try to use unique icons, due to unsupported size, they are resized to 24x24 which makes them looking bad. Oleg decided to select 32x32 size because he thought 16x16 would be too small.